pwiz_tools/Skyline/CommandLine.cs

```diff
@@ -1303,6 +1303,7 @@ private bool ImportSearchInternal(CommandArgs commandArgs, ref SrmDocument doc)
 
             // Build library
             var builder = import.GetLibBuilder(doc, commandArgs.Saving ? commandArgs.SaveFile : commandArgs.SkylineFile, commandArgs.IncludeAmbiguousMatches);
+            builder.PreferEmbeddedSpectra = commandArgs.PreferEmbeddedSpectra;

```

Comment:
This file seems to be missing special handling for the spectrum file missing error. You need a message that tells the user to relocate the raw data or use --import-search-prefer-embedded-spectra